modules/gridNMBidAdapter.js

```diff
@@ -91,11 +94,32 @@ export const spec = {
         }
       }
 
+      const video = utils.deepAccess(bid, 'mediaTypes.video') || {};
+      const paramsVideo = Object.assign({}, params.video);
+      VIDEO_KEYS.forEach((key) => {
+        if (!(key in paramsVideo) && key in video) {
+          paramsVideo[key] = video[key];
+        }
+      });
+
+      if (!paramsVideo.size && video.playerSize && video.playerSize.length === 2) {
+        paramsVideo.size = video.playerSize.join('x');
+      }
+
+      if (!('mind' in paramsVideo) && 'minduration' in video) {
+        paramsVideo.mind = video.minduration;
+      }
+      if (!('maxd' in paramsVideo) && 'maxduration' in video) {
+        paramsVideo.maxd = video.maxduration;
+      }
```

Comment:
This looks fine but curious as to why 'mind' and 'maxd' are not included in VIDEO_KEYS and added to paramsVideo in the loop above? It doesn't seem like there is special logic occurring to build as there is in 'size'

Reply:
It was made, because in mediaTypes.video sizes represents as minduration and maxduration, but our endpoint wait it as mind and maxd